vendor/bat-native-ledger/src/bat/ledger/internal/api/api_parameters.h

```diff
@@ -0,0 +1,40 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVELEDGER_API_API_PARAMETERS_H_
+#define BRAVELEDGER_API_API_PARAMETERS_H_
+
+#include "bat/ledger/ledger.h"
+#include "bat/ledger/mojom_structs.h"
```

Comment:
Isn't this already included in `ledger.h`?

Reply:
yup, fixed